src/overlays/actors/ovl_En_Nb/z_en_nb_cutscene_data.c

```diff
@@ -0,0 +1,174 @@
+#include "z_en_nb.h"
+
+#include <z64cutscene_commands.h>
+
+static CutsceneData D_80AB431C[] = {
+
+    CS_BEGIN_CUTSCENE(27, 3011),
+    CS_UNK_DATA_LIST(0x00000020, 1),
+    CS_UNK_DATA(0x00010000, 0x0BB80000, 0x00000000, 0x00000000, 0xFFFFFFFC, 0x00000002, 0x00000000, 0xFFFFFFFC,
+                0x00000002, 0x00000000, 0x00000000, 0x00000000),
+    CS_NPC_ACTION_LIST(31, 5),
+    CS_NPC_ACTION(0x0001, 0, 501, 0x0000, 0x0000, 0x0000, 0, 216, 0, 0, 216, 0, 0.0f, 0.0f, 0.0f),
```

Comment:
I think the formatting here is "incorrect" in the sense that it was modified by the formatter. We usually disable clang-format around cutscene data for now, to avoid this issue.